content/blog/2023/questioning-my-approach-to-advice.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Questioning My Approach to Advice"
+date: 2023-01-20
+draft: false
+---
+
+Good advice is very difficult to give. This is because it is inherently context dependent. Advice is inextricably linked with the lived experience of the person giving the advice. No matter how well conceived or written, the value of advice is limited by its applicability to your situation.
+
+Given how unlikely it is to produce advice that is helpful, what is someone like me with a blog supposed to do? I see three options: keep giving advice anyways; stop giving advice at all; or make the advice context independent.
+
+Ignoring the problems with advice and barreling through is the easiest approach. It's not a problem if I don't acknowledge it 😉. More seriously, I think there is still some limited upside to this approach. While the advice you give is still going to be context dependent, if your audience is big enough then at least some people's context will intersect with yours enough to make the advice useful. Overall though, this approach leaves a lot of equity on the table.
+
+Another approach that takes the easy way out is just refraining from giving advice at all. In other words sharing your experience but refusing to extrapolate any lessons from it. This approach does solve the problem of giving advice that doesn't apply to others, but it also throws the baby out with the bath water. Drawing no lessons when I share my experience feels like a step too far.
+
+The final approach suggests that we should make the advice context independent. Doing this would solve the fundamental problem with most advice. One way we could do this is by making our advice more generic. In this way we increase the amount of people who it would apply to. However, making advice generic very quickly turns it into useless truisms. If that doesn't work then what will?
```

Comment:
don't end sentence with prepostion
```suggestion
The final approach suggests that we should make the advice context independent. Doing this would solve the fundamental problem with most advice. One way we could do this is by making our advice more generic. In this way we increase the amount of people to whom it would apply. However, making advice generic may turn it into useless truisms. If that doesn't work then what will?
```